src/Node.ts

```diff
@@ -163,21 +163,19 @@ const emptyChildren: Collection<any> = new Collection();
 let idCounter = 1;
 
 // create all the events here
-interface NodeEventMap {
+type NodeEventMap = GlobalEventHandlersEventMap & {
```

Comment:
Wow. Thanks for that `GlobalEventHandlersEventMap`. Did know it exists. Super helpful.